src/field.rs

```diff
@@ -87,7 +87,7 @@ impl<F: Field> Element<F> {
         // From Fermat's little theorem.
         // TODO: Use a faster method, like the one described in "Fast Modular Reciprocals".
         // Or just wait for https://github.com/rust-num/num-bigint/issues/60
-        self.exp(&-Self::from(2u8))
+        self.exponentiation(&-Self::from(2u8))
```

Comment:
Renamed to `exponentiation` for notation consistency away from references to `Expression`.